src/mapEditDialog.cpp

```diff
@@ -66,7 +66,37 @@ QString MapEditDialog::getMusic() const
     return m_ui->lineEditMusic->text();
 }
 
-void MapEditDialog::onChipsetBrowse()
+bool MapEditDialog::inputsAreValid(QString* errorMessage)
+{
+
+    if (m_ui->lineEditMapName->text().isEmpty()) {
+        if (errorMessage)
```

Comment:
Correct, yet definitely not a big fan of doing it this way to check pointer's validity.

Unless there a C++ operator overloading for checking "truthness" of an object, I’d write `if (ptr != nullptr)` or even `if (nullptr != ptr)`.

But I am no more in charge here. You guys do what you want.

Reply:
Agreed !

I think nothing except a **bool** should be directly-tested in if...